ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -391,17 +350,46 @@ The workflow diagram includes the following elements:
 - [APPLY CHANGES]  
     Click this button to save all changes made to the trigger(s).
 
+## Workflow &ndash; Compare workflow versions
+
+*Process Orchestration > Workflows > Tab OVERVIEW > Select workflow > Select version > Button Points > Menu entry Show changes*
+
+*Process Orchestration > Workflows > Tab OVERVIEW > Select workflow > Select version > Click COMPARE button*
+
+  ![Workflow](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowComparison.png "[Workflow]")  
+
+**Workflows**
+
+The *Workflow* window displays two versions of a workflow configuration.
+
+The following fields and functions are available in this window:
+
+- *Workflow*   
+    Click this drop-down list to select the workflow you want to use for comparison.
+
+- ![Swap](../../Assets/Icons/Swap.png "[Swap]") [SWAP VERSIONS]   
+    Click this button to swap the versions displayed.
+
+- ![Format code](../../Assets/Icons/Swap.png "[Format code]") [FORMAT CODE]   
+    Click this button to compare the lines of code and puts them in the same order, so that order differences are no longer highlighted.
```

Comment:
```suggestion
    Click this button to compare the lines of code and put them in the same order, so that order differences are no longer highlighted.
```